src/main/java/dev/slimevr/config/AutoboneConfig.java

```diff
@@ -0,0 +1,28 @@
+package dev.slimevr.config;
+
+public class AutoboneConfig {
```

Comment:
Minor gripe, the `B` not being capitalized, this change will require renaming the file too but GitHub doesn't let you do that -w-
```suggestion
public class AutoBoneConfig {
```